.github/workflows/ci.yml

```diff
@@ -494,16 +494,27 @@ jobs:
         uses: mamba-org/setup-micromamba@v1
         with:
           environment-file: modflow6/environment.yml
+          create-args: >-
+            netcdf-fortran
           init-shell: >-
             bash
             powershell
           cache-environment: true
           cache-downloads: true
 
+      - name: Update PKG_CONFIG_PATH
+        working-directory: modflow6
+        run: |
+          if [ "$RUNNER_OS" == "Linux" ]; then
+            echo "PKG_CONFIG_PATH=/home/runner/micromamba/envs/modflow6/lib/pkgconfig:$PKG_CONFIG_PATH" >> $GITHUB_ENV
```

Comment:
I think this could be done platform-independently via `CONDA_PREFIX` env var or `environment-path` [output](https://github.com/mamba-org/setup-micromamba/blob/bdff0b7f317300a7c04a9050b17168c9a96a2a9a/action.yml#L91)